src/main/resources/com/github/terma/jenkins/githubprcoveragestatus/Configuration/help-sonarToken.html

```diff
@@ -0,0 +1,4 @@
+<div>
+    Access token for Sonar instance(Optional)
+    <a href="https://docs.sonarqube.org/display/SONAR/User+Token">Sonar User Token</a>
```

Comment:
As soon as we have a few auth options for Sonar ```anonym|login/password|token``` it will be good to provide information about how they work together